README.md

```diff
@@ -1 +1,79 @@
-# libuast
\ No newline at end of file
+# libuast
+
+## Installation
+
+### 1. Build/Install C++ API
+
+```
+cmake .
+make
+make install
+```
+
+### 2.Install python bindings
+
+```
+## move to bindings
+cd bindings
+cmake .
+make
+make install-python
+```
+
+## Usage
+
+### C++
```

Comment:
It may be better to refererence the example C++ file directly, since it'll probably will change in the future and that'll prevent us from having to keep these changed synced.